### PR TITLE
Add ABE and EFGA

### PR DESCRIPTION
**Pull request by patricksanders**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /public
 *.swp
+.DS_Store
```
Comment (brimstone) [inline .gitignore:3]: :+1:

File: themes/agency/static/css/agency.css [hunk 2]
```diff
@@ -1022,3 +1022,4 @@ img::-moz-selection {
 body {
     webkit-tap-highlight-color: #282260;
 }-
+
```
Comment (brimstone) [inline themes/agency/static/css/agency.css]: haaaay

**Review by brimstone — APPROVED** [review]
I had been normalizing the images to 360px wide, but this is fine I suppose. Merge if you don't want to resize them.

File: themes/agency/static/css/agency.css (hunk 2 above)
Comment (patricksanders) [reply themes/agency/static/css/agency.css]: 🙈

**Pull request merged**